coala_quickstart/generation/Utilities.py

```diff
@@ -0,0 +1,15 @@
+def comma_split(response):
+    """
+    Splits the comma-separated string into a list.
+
+    :param response:
+        Comma-separated string.
+    :return:
+        A list of comma-separated elements in the string.
+    """
+    result = []
+    for item in response.replace(" ", ",").split(","):
+        if len(item):
+            result.append(item)
+
+    return result
```

Comment:
Hmm, yes. I should do that. `src\,tests` will be split when it shouldn't be. :+1: